programs/benchmark/Benchmark.cpp

```diff
@@ -548,11 +548,13 @@ class Benchmark : public Poco::Util::Application
             json_out << double_quote << connections[i]->getDescription() << ": {\n";
             json_out << double_quote << "statistics" << ": {\n";
 
-            print_key_value("QPS", info->queries / info->work_time);
-            print_key_value("RPS", info->read_rows / info->work_time);
-            print_key_value("MiBPS", info->read_bytes / info->work_time);
-            print_key_value("RPS_result", info->result_rows / info->work_time);
-            print_key_value("MiBPS_result", info->result_bytes / info->work_time);
+            double seconds = info->work_time / concurrency;
+
+            print_key_value("QPS", info->queries.load() / seconds);
+            print_key_value("RPS", info->read_rows / seconds);
+            print_key_value("MiBPS", info->read_bytes / seconds);
+            print_key_value("RPS_result", info->result_rows / seconds);
+            print_key_value("MiBPS_result", info->result_bytes / seconds);
```

Comment:
In labels its MiB, but output is bytes
```suggestion
            print_key_value("MiBPS", info->read_bytes / seconds / 1048576);
            print_key_value("RPS_result", info->result_rows / seconds);
            print_key_value("MiBPS_result", info->result_bytes / seconds / 1048576);
```